discovery-provider/src/models/models.py

```diff
@@ -261,10 +261,25 @@ class Track(Base):
         viewonly=True,
     )
 
+    _owner = relationship(
```

Comment:
this seems like it would almost just do what we do in a lot of places with the `with_users` arg (e.g. tracks returning users already).
It feels like this is effectively a no-op in terms of perf since we're going to do the join down the line regardless, but maybe we can look into cleaning that up to improve the number of round-drips?

Reply:
It does seem like most places we get the user anyway, but I agree we should be able to avoid manual joins. I'll poke around a little but I don't want to scope creep it into this PR I don't think

Reply:
agreed we shouldn't scope creep. i don't think anything super critical path relies on /tracks anymore, so this change is totally fine. benefit being that we can eventually rally around it and move off of the `with_users` pattern

Reply:
i think everywhere in the client we call with_users on by default
https://github.com/AudiusProject/audius-client/blob/f6c9d565d7f93a668753aaff7b25e9070065a507/src/services/AudiusBackend.js#L619

Reply:
A brief look at line 151 of `get_tracks.py` shows that we call `get_users_by_id` which populates user metadata (`populate_user_metadata` in `query_helpers.py`) and that looks much more heavy duty than what's in the user table with this relationship unfortunately.

Reply:
hypothetically couldn't we do this relationship join and keep the ID and then call into populate_user_metadata for each of the tracks' users?

Reply:
Hmm true, we could skip the call into `get_unpopulated_users`